src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -59,6 +62,23 @@ public void testNewQuestionFrame() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+
+        ______TS("MCQ: Check UI after cancelling and add new MCQ question again");
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(-QUESTION_INDEX_ONE);
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
+
+        feedbackEditPage.clickDiscardChangesLinkForNewQuestion();
+        feedbackEditPage.waitForConfirmationModalAndClickOk();
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("MCQ");
+        assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
+        assertTrue(feedbackEditPage.isElementEnabled("mcqGenerateForSelect--1"));
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(-QUESTION_INDEX_ONE); //Make the generate options checkbox unchecked
```

Comment:
Why not just define the new question index as `-1` like what we do in JS file `instructorFeedbackEdit.js`.
``` javascript
const NEW_QUESTION = -1;
```